transformations/grapheme_to_phoneme_transformation/transformation.py

```diff
@@ -0,0 +1,78 @@
+import random
+import re
+import spacy
+import string
+import pronouncing
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+
+
+def grapheme_phoneme(grapheme):
+    '''
+    converts each word to phonems. If there are multiple pronounciation of a
+    word, only the first prononciation is taken. Stress information from each
+    word is removed.
+    '''
+    phoenems = pronouncing.phones_for_word(grapheme)
+
+    if len(phoenems)> 0:
+        phoenem_without_stress = ''.join([x for x in phoenems[0] if x.isalpha()]).lower()
```

Comment:
Thanks for your changes! One comment here - why not use the remaining pronunciations to create multiple paraphrases?

Reply:
Hi, Thank you for the comments. The reason I considered only one pronunciation is that the differences in pronunciations were mostly because of stress applied in different syllables(phones) rather than syllables themselves in most of the cases.